iChef-WebApi/src/main/java/com/kitchen/iChef/Controller/Model/Request/RecipeRequest.java

```diff
@@ -0,0 +1,51 @@
+package com.kitchen.iChef.Controller.Model.Request;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.validation.Valid;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+import java.util.List;
+
+@Getter
+@Setter
+@ToString
+public class RecipeRequest {
+    @NotBlank(message = "The title of the recipe is not valid!")
+    private String title;
+
+    @NotBlank(message = "The utensils of the recipe is not valid!")
+    private String utensils;
+
+    @NotBlank(message = "The steps of the recipe is not valid!")
+    private String steps;
+
+    @Min(value = 1, message = "The rating of the recipe is not valid!")
+    private Float rating;
```

Comment:
You don't want to set the rating when you create a recip. It should be added separately.